app/helpers/form_helper.rb

```diff
@@ -1,9 +1,33 @@
 module FormHelper
-  def text_f(f, attr, options = {})
+  def text_f(f, attr, options = {}, btn_options = {})
+    btn_group = btn_options.delete(:button_group)
+    btn_class = btn_options.delete(:class) || 'col-md-4 select-margin'
+    label = btn_options.delete(:label)
+    documentation = btn_options.delete(:documentation)
     field(f, attr, options) do
       addClass options, "form-control"
       options[:focus_on_load] ||= attr.to_s == 'name'
-      f.text_field attr, options
+      text_field =
+        content_tag :div, :class => btn_group ? "col-md-8" : "" do
+          f.text_field attr, options
+        end
+      if btn_group
+        button =
+          if documentation
+            content_tag :div, :class => btn_class do
+              content_tag :span, class: 'input-group-btn' do
+                documentation_button(label)
```

Comment:
Can this just be passed into the helper, so this helper doesn't need some special knowledge of documentation buttons?  Can it simply use button_input_group?

The helper's becoming overly complex.